tests/integration/bundles/kfp_latest_edge.yaml.j2

```diff
@@ -1,13 +1,35 @@
 bundle: kubernetes
 name: kubeflow-pipelines
 applications:
-  argo-controller:         { charm: ch:argo-controller, channel: latest/edge, scale: 1 }
+  argo-controller:         { charm: ch:argo-controller, channel: latest/edge, scale: 1, trust: true }
   metacontroller-operator: { charm: ch:metacontroller-operator, channel: latest/edge, scale: 1, trust: true }
   minio:                   { charm: ch:minio, channel: latest/edge,       scale: 1 }
-  kfp-db:                  { charm: ch:mysql-k8s, channel: 8.0/stable, scale: 1, constraints: mem=2G}
+  kfp-db:                  { charm: ch:mysql-k8s, channel: 8.0/stable, scale: 1, constraints: mem=2G, trust: true }
+  mlmd:                    { charm: ch:mlmd, channel: latest/edge, scale: 1 }
+  envoy:                   { charm: ch:envoy, channel: latest/edge, scale: 1 }
+  kubeflow-profiles:       { charm: ch:kubeflow-profiles, channel: latest/edge, scale: 1, trust: true }
+  istio-ingressgateway:
+    charm: istio-gateway
+    channel: latest/edge
+    scale: 1
+    options:
+      kind: ingress
+    trust: true
+  istio-pilot:
+    charm: istio-pilot
+    channel: latest/edge
+    scale: 1
+    options:
+      default-gateway: kubeflow-gateway
+    trust: true
+  kubeflow-roles:
+    charm: kubeflow-roles
+    channel: latest/edge
+    scale: 1
+    trust: true
 {%- if local_build == false %}
   kfp-api:                 { charm: ch:kfp-api, channel: latest/edge, scale: 1, trust: true}
-  kfp-profile-controller:  { charm: ch:kfp-profile-controller, channel: latest/edge, scale: 1, trust: true }
+  kfp-metadata-writer:     { charm: ch:kfp-metadata-writer, channel: latest/edge, scale: 1, trust: true}
```

Comment:
Apart from seeing kfp come alive, do we have any tests that prove the metadata writer is working correctly?  Maybe running a pipeline and then exporting some metadata to prove it was logged correctly?  Or a manual test that shows in the UI where to see the data?

Reply:
We don't have any, but we can definitely add one or two test cases for the metadata-writer, though that seems to belong in a different PR.

Reply:
Agreed.  added as #384 and resolving this